DashAI/front/src/api/component.ts

```diff
@@ -0,0 +1,52 @@
+import api from "./api";
+import type { IDataloader } from "../types/dataloader";
+import type { ITask } from "../types/task";
+
+interface componentInput {
```

Comment:
Do you agree if we call this interface `componentQuery`?

```suggestion
interface componentQuery{
```